Library/Homebrew/utils.rb

```diff
@@ -517,3 +517,18 @@ def migrate_legacy_keg_symlinks_if_necessary
   end
   FileUtils.rm_rf legacy_pinned_kegs
 end
+
+def puts_hash(hash, indent: 0)
+  return hash unless hash.is_a? Hash
+  hash.each do |key, value|
+    printf "#{" " * indent*2}#{key}:"
```

Comment:
Sure — I think `" " * (indent * 2)` would be more readable, since the `*` operator is doing two different jobs — while order technically doesn't matter, it's a somewhat confusing thing to have to think through IMO.